src/Fuel-Platform-Core/FLPlatform.class.st

```diff
@@ -95,27 +95,11 @@ FLPlatform class >> extensionProtocolName [
 FLPlatform class >> fixMetacello [
 ]
 
-{ #category : #accessing }
-FLPlatform class >> hacksCategoryName [
-	^ 'FuelPlatformHacks'
-]
-
 { #category : #testing }
 FLPlatform class >> isResponsibleForCurrentPlatform [
 	^ self subclassResponsibility
 ]
 
-{ #category : #'private-convenience' }
-FLPlatform class >> removeModifications [
```

Comment:
Fuel has all this code because it is supposed to be working on all old version of Pharo and all current and many (all?) versions of Squeak. This means of course that this "feature" at some point will cost a lot.

 Already this change is not how to do it, there are subclasses for FLPlatForm... 

And no, I do not think this makes any sense for something so dependend on implementation details to try to be compatible like that.